before_install_linux.sh

```diff
@@ -16,8 +16,12 @@ if [ ! -d /usr/include/freetype ]; then
 fi
 
 if [ -v IMAGEMAGICK_VERSION ]; then
-  wget http://www.imagemagick.org/download/releases/ImageMagick-${IMAGEMAGICK_VERSION}.tar.xz
-  tar -xf ImageMagick-${IMAGEMAGICK_VERSION}.tar.xz
+  if wget https://github.com/ImageMagick/ImageMagick/archive/${IMAGEMAGICK_VERSION}.tar.gz; then
+    tar -xzf ImageMagick-${IMAGEMAGICK_VERSION}.tar.gz
+  else
+    wget http://www.imagemagick.org/download/releases/ImageMagick-${IMAGEMAGICK_VERSION}.tar.xz
+    tar -xf ImageMagick-${IMAGEMAGICK_VERSION}.tar.xz
```

Comment:
Is there not a `gz` version of this in both places? If we could get the same file type, we could move the `tar` outside of the `if` statement.

Reply:
There is no `.tar.gz` for the 6.7.7 build and GitHub only provides a `.tar.gz` download.